amity.py

```diff
@@ -529,8 +529,33 @@ def print_unallocated(self, *args):
         else:
             print("Please input valid text file: eg unallocated")
 
-    def print_room(self):
-        pass
+    
+    def print_room(self, room_name):
+        if isinstance(room_name, str):
+            all_rooms =[]
+            allocated_office_names =[room for room in self.office_allocations]
+            allocated_living_space_names =[room for room in self.living_space_allocations]
+            for room in self.offices:
+                all_rooms.append(room["name"])
+            for room in self.living_spaces:
```

Comment:
It would help to consider combining certain set attributes when it comes to data relationship, which will aid in writing short, re-useable  and cleaner code in the future.